src/components/Tables/FilterAccordion.tsx

```diff
@@ -56,6 +55,20 @@ const Spacer = styled.div`
     flex-grow: 1;
 `;
 
+const StyledAccordion = styled(Accordion)`
```

Comment:
can Styling be removed and replaced with this new StyledAccordion, or are there some things that don't work?

Reply:
Moved those specific to the accordion styling to `StyledAccordion`
However, some stuff like `height` has to be fixed on top otherwise the table header will grow